src/main/resources/component-web.conf

```diff
@@ -71,6 +71,11 @@ http {
     # agents caring about that). A detailed description of P3P can be found here: http://en.wikipedia.org/wiki/P3P
     addP3PHeader = true
 
+    # Determines the content security policy, i.e. which internet resources to load from which locations. By default,
+    # an unsecure "allow everything" policy is set. A more secure value could be "script-src 'self'". A description of
```

Comment:
shouldn't we also recommed inline-insecure-whatever?

Reply:
done